modules.d/99squash/init-squash.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/sh
```

Comment:
```suggestion
#!/bin/sh
```

Reply:
Thanks for the review, will fix it to /bin/sh. This part is being included in the initramfs, so I used /usr/bin/sh as the squash module setup will organize the initramfs that way and didn't create /bin and /sbin as symlink to /usr/xbin, that looks no so good after a second thought.